src/tmx/TmxCore/CMakeLists.txt

```diff
@@ -12,7 +12,9 @@ TARGET_INCLUDE_DIRECTORIES( ${PROJECT_NAME} PUBLIC
                             ${MYSQL_INCLUDE_DIRS} ${MYSQLCPPCONN_INCLUDE_DIRS} )
 TARGET_LINK_LIBRARIES ( ${PROJECT_NAME} PUBLIC ${TMXAPI_LIBRARIES} )
 TARGET_LINK_LIBRARIES ( ${PROJECT_NAME} PUBLIC ${MYSQL_LIBRARIES} ${MYSQLCPPCONN_LIBRARIES} )
-TARGET_LINK_LIBRARIES ( ${PROJECT_NAME} PUBLIC pthread m rt )
+TARGET_LINK_LIBRARIES ( ${PROJECT_NAME} PUBLIC libasn_j2735 pthread m rt )
```

Comment:
Lets link ::carma_j2735 directly instead of the alias

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/654/commits/ef1238953d99aa1f7aa13b59a7133c1131039ee5